code/modules/mob/new_player/new_player.dm

```diff
@@ -251,6 +251,8 @@
 		return 0
 	if((job.current_positions >= job.total_positions) && job.total_positions != -1)
 		if(job.title == "Assistant")
+			if(client.player_age <= 14) //Newbies can always be assistants

```

Comment:
This is going to runtime if you don't check if player_age is a number, because it's a text string if the db isn't connected